src/pathway/GPU-kernel.cuh

```diff
@@ -13,6 +13,10 @@
 #define THREAD_ID (threadIdx.x)
 #define GLOBAL_ID (THREAD_ID + NT * blockIdx.x)
 #define BLOCK_ID  (blockIdx.x)
+#
+#ifndef UINT32_MAX
+#define UINT32_MAX  (0xffffffff)
+#endif
 
```

Comment:
At first, I'm not so familiar with C++. (I'm using pure C, usually)
I thought it may or should be defined in stdint.h, but compiler complained that UINT_MAX is not defined in AWS environment.
So I just fixed it, and I'm thinking it should be no difference to define UINT_32 as in code at least in behavior level. Is it right?

reference
* http://stackoverflow.com/questions/1471353/whats-the-c-equivalent-of-uint32-max
* https://bugzilla.mozilla.org/show_bug.cgi?id=673556
> The c99 standard says:
> 
> C++ implementations should define these macros only when __STDC_LIMIT_MACROS
> is defined before <stdint.h> is included
> 
> and "these macros" are the limit ones like UINT32_MAX.